### PR TITLE
News + Contact

### PR DESCRIPTION
**Pull request by davidwessman**
Fixing format issue with contact and news.
Added some controller unit-tests for news.

### CODE REVIEW

File: app/controllers/news_controller.rb [hunk 1]
```diff
@@ -17,25 +17,24 @@ def edit
   end
 
   def create
-    @news.profile = current_user.profile        
+    @news.profile = current_user.profile
     @news.save!
-    redirect_to @news
+    redirect_to news_path(@news)
```
Comment (henrikssn) [inline app/controllers/news_controller.rb]: Security fixes, yay! :+1:

**Comment by henrikssn** [comment]
LGTM

Ship it!

**Comment by davidwessman** [comment]
Aaa, now we have to merge with current state ;)

**Comment by henrikssn** [comment]
That is why small PRs are yay and big are nay :)

**Comment by davidwessman** [comment]
Hehe yea.
I have a thing for touching too much code at once :( 
Still learning this big dev thingy.

**Comment by SirDirac** [comment]
This PR was deployed to Production. Reference: <a href='http://samson.fsektionen.se/projects/web/deploys/122' target='_blank'>d92d346</a>

**Pull request merged**